ci/builders/mac_unopt.json

```diff
@@ -111,6 +111,109 @@
                     "type": "local"
                 }
 
+            ]
+        },
+        {
+            "archives": [
+                {
+                    "base_path": "out/host_debug_unopt_arm64/zip_archives/",
+                    "type": "gcs",
+                    "include_paths": [
+                    ],
+                    "name": "host_debug_unopt_arm64"
```

Comment:
Is the only purpose of this build to validate it builds correctly?

Reply:
For now, until we can run tests.

Reply:
I should have said:

We're not ready to run host unit tests on CI for reasons in https://github.com/flutter/flutter/issues/103386. It seems close.

This is running tests on iOS simulator though.